src/services/recordService.ts

```diff
@@ -160,125 +159,66 @@ const recordService = {
       );
   },
 
-  searchWithFallbackIfNeeded(
-    ownerId: string,
-    countOnly: boolean,
-    params: Params = {}
-  ): Promise<any[]> {
-    const searchParams = { ...params };
-
-    const appDataFlag = taggingUtils.generateAppDataFlagTag();
-    if (!params?.tags?.includes(appDataFlag)) {
-      searchParams.exclude_flags = [appDataFlag];
+  /**
+   * Encrypt a tag or a tag group with the user's tag encryption key
+   *
+   * A tag group starts with an opening bracket (_(_) and ends with a closing bracket (_)_) and
+   * includes a comma separated list of tags
+   * @param tek The tag encryption key of the user
+   * @param tag The tag or tag group to encrypt
+   * @returns The encrypted tag or tag group
+   */
+  async encryptTag(tek: SymKey, tag: string) {
```

Comment:
Just a small thing - since iOS and KMP do not speak of `tek` anymore and writing it out, you consider this here as well - it would be great if we could use nearly the same terminology, from my perspective.